target/linux/ramips/dts/mt7628an_tplink_archer-c20-v4.dts

```diff
@@ -76,8 +76,8 @@
 	mtd-mac-address-increment = <(-2)>;
 };
 
-&ethernet {
-	mediatek,portmap = "wllll";
+&esw {
+	mediatek,portmap = <0x3e>;
```

Comment:
Ah, so this already overwrites tplink_8m. This convinces me in dropping the statement in tplink_8m entirely.